src/generated/xinput.rs

```diff
@@ -15404,51 +15850,6 @@ impl TryParse for EventForSend {
         Ok((result, &value[32..]))
     }
 }
-impl From<DeviceValuatorEvent> for EventForSend {
-    fn from(value: DeviceValuatorEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceKeyPressEvent> for EventForSend {
-    fn from(value: DeviceKeyPressEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceFocusInEvent> for EventForSend {
-    fn from(value: DeviceFocusInEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceStateNotifyEvent> for EventForSend {
-    fn from(value: DeviceStateNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceMappingNotifyEvent> for EventForSend {
-    fn from(value: DeviceMappingNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<ChangeDeviceNotifyEvent> for EventForSend {
-    fn from(value: ChangeDeviceNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceKeyStateNotifyEvent> for EventForSend {
-    fn from(value: DeviceKeyStateNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DeviceButtonStateNotifyEvent> for EventForSend {
-    fn from(value: DeviceButtonStateNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
-impl From<DevicePresenceNotifyEvent> for EventForSend {
-    fn from(value: DevicePresenceNotifyEvent) -> Self {
-        Self(<[u8; 32]>::from(value))
-    }
-}
```

Comment:
This looks like another bunch of API being lost. Something around `eventstruct` handling (and yes, this is the only use of `eventstruct` in the XML)

Reply:
Fixed